angular/src/lib/select/select.component.ts

```diff
@@ -69,44 +70,154 @@ export class SelectChange {
       [cdkConnectedOverlayOpen]="overlayOpen"
       [cdkConnectedOverlayWidth]="this.anchorWidth"
       [cdkConnectedOverlayOffsetY]="6"
-      [cdkConnectedOverlayPanelClass]="'md-select__dropdown'"
+      [cdkConnectedOverlayPanelClass]="['md-select__dropdown', overlayClass]"
       [cdkConnectedOverlayHasBackdrop]="true"
       [cdkConnectedOverlayBackdropClass]="'cdk-overlay-transparent-backdrop'"
       (backdropClick)="close()"
       (detach)="close()"
     >
-      <md-list
-        #panel
-        role='listbox'
-        tabType="vertical"
-        [ngClass]="listClass"
-      >
-        <ng-content></ng-content>
-      </md-list>
+
+        <div
+          *ngIf="filter"
+          class="md-select__filter md-input-group md-search-input md-search-input--pill"
```

Comment:
@pauljeter Couple thoughts in the context of inputRework feature branch...
1) md-search-input is no longer as a class nor is --pill
2) semantically odd that md-input-group and md-search-input would be on the same line.  
3) we could prevent any current rework by applying those style to the md-select__filter class (expedite input merge)